docs/ref/models/conditional-expressions.txt

```diff
@@ -42,9 +42,11 @@ We'll be using the following model in the subsequent examples::
 A ``When()`` object is used to encapsulate a condition and its result for use
 in the conditional expression. Using a ``When()`` object is similar to using
 the :meth:`~django.db.models.query.QuerySet.filter` method. The condition can
-be specified using :ref:`field lookups <field-lookups>` or
-:class:`~django.db.models.Q` objects. The result is provided using the ``then``
-keyword.
+be specified using :ref:`field lookups <field-lookups>`,
+:class:`~django.db.models.Q` objects, or
+:class:`~django.db.models.Expression` objects that have an
+``output_field`` that is a :class:`~django.db.models.BooleanField`.
+The result is provided using the ``then`` keyword.
```

Comment:
It should be noted that support for conditional expressions where only added in 3.0.

Reply:
I updated the documentation but I'm not too sure how it should be done. Let me know about you think about the latest changes or if I should add a new paragraph under a `.. versionadded:: 3.0` or if that is unnecessary in this context.

Reply:
add a new paragraph under a .. versionadded:: 3.0